lib/init.coffee

```diff
@@ -1,61 +1,59 @@
-{BufferedProcess, CompositeDisposable} = require 'atom'
+{ BufferedProcess, CompositeDisposable } = require 'atom'
+helpers = require 'atom-linter'
 
 module.exports =
-  config:
-    executablePath:
-      type: 'string'
-      title: 'The path where Roodi is located'
-      default: 'roodi'
   activate: ->
-    console.log 'activate linter-roodi'
+    console.log 'activate linter-roodi' if atom.inDevMode()
     @subscriptions = new CompositeDisposable
     @subscriptions.add atom.config.observe 'linter-roodi.executablePath',
-      (executablePath) =>
+      (executablePath) ->
         @executablePath = executablePath
   deactivate: ->
     @subscriptions.dispose()
   provideLinter: ->
     provider =
+      name: 'roodi'
       grammarScopes: ['source.ruby', 'source.ruby.rails', 'source.ruby.rspec']
       scope: 'file' # or 'project'
-      lintOnFly: true # must be false for scope: 'project'
-      lint: (textEditor) =>
-        return new Promise (resolve, reject) =>
+      lintsOnChange: true # must be false for scope: 'project'
+      lint: (textEditor) ->
+        return new Promise (resolve, reject) ->
           filePath = textEditor.getPath()
-          output = ""
+          output = ''
           process = new BufferedProcess
             command: @executablePath
-            args: [filePath, '--json']
```

Comment:
Huh, was the `--json` simply being ignored?

Reply:
I did check the roodi source code and it has no --json option.